openmc/model/model.py

```diff
@@ -66,6 +91,19 @@ def __init__(self, geometry=None, materials=None, settings=None,
         if plots is not None:
             self.plots = plots
 
+        self.chain_file = chain_file
+        self.fission_q = fission_q
+
+        if materials is None:
+            mats = self.geometry.get_all_materials().values()
+        else:
+            mats = self.materials
+        self._materials_by_id = {mat.id: mat for mat in mats}
+        self._materials_by_name = {mat.name: mat for mat in mats}
```

Comment:
Ah, yes, thanks! I meant to leave a note about that in the docs. My original intent was to just leave a documentation ``.. note::`` in the ``rotate_cells``, ``translate_cells``, etc, methods that accessing the correct cell is not guaranteed for non-unique names.

However, right now in thinking about it I think the approach I will take is to store a list of all the cells that share a given name and operate on all of them. This could be super useful as a user by naming all things that need to be moved (e.g.) the same thing.  This will still have a documentation note to explain the behavior.